Utils/Dataflow/pyDKB/common/hdfs.py

```diff
@@ -137,6 +181,14 @@ def listdir(dirname, mode='a'):
         mode    -- 'a': list all objects
                    'f': list files
                    'd': list subdirectories
+    
+    :param dirname: name of HDFS directory
+    :type dirname: string
+    :param mode: type of a list of files
+    :type mode: string
+    
+    :return: a list of files' names/subdirectories inside HDFS
```

Comment:
...in HDFS directory.
"Inside HDFS" looks a bit too broad and can make one to believe it is about all files/subdirectories.